modules/tinymce/src/themes/silver/test/ts/browser/window/SilverDialogApiAccessTest.ts

```diff
@@ -93,6 +103,26 @@ describe('browser.tinymce.themes.silver.window.SilverDialogApiAccessTest', () =>
           'newData: Init Value'
         ]));
       });
+
+      it('TINY-9528: fullscrenn toggle should apply the correct class', () => {
+        const editor = hook.editor();
+        DialogUtils.open(editor, dialogSpec, test.params);
+
+        const dialog = UiFinder.findIn(TinyDom.fromDom(document), '.tox-dialog').getOrDie();
+        const dialogHasClass = (className: string) => dialog.dom.classList.contains(className);
+
+        assert.isFalse(dialogHasClass('tox-dialog--fullscreen'), 'before toggle dialog should not have class tox-dialog--fullscreen');
+
+        Mouse.clickOn(SugarBody.body(), 'button:contains("Toggle fullscreen")');
```

Comment:
There is a `TInyUiActions` module that can be used to click on dialog things. See if you can use that.

Reply:
done